test/intl402/Segmenter/constructor/constructor/options-granularity-abrupt-throws.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-intl.segmenter
+description:
+    Return abrupt completion from GetOption localeMatcher
+info: |
+    Intl.Segmenter ([ locales [ , options ]])
+
+    13. Let granularity be ? GetOption(options, "granularity", "string", « "grapheme", "word", "sentence" », "grapheme").
+
+    GetOption ( options, property, type, values, fallback )
+    1. Let value be ? Get(options, property).
+features: [Intl.Segmenter]
+---*/
+
+
+var options = {};
+Object.defineProperty(options, 'localeMatcher', {
```

Comment:
Should this be `granularity `?

Reply:
> Should this be `granularity `?

sorry. My bad. Must be too tired last night and forget to change it after copy the file.